src/routes/docs/tooling/assistant/appwriter/+page.markdoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: article
+title: The Appwriter
+description: Learn about the custom Appwriter mechanical keyboard and its specifications
+---
+
+{% only_dark %}
+![The Appwriter](/images/docs/keyboard/dark/appwriter.png)
+{% /only_dark %}
+
+{% only_light %}
+![The Appwriter](/images/docs/keyboard/appwriter.png)
+{% /only_light %}
+
+The Appwriter is an exclusive mechanical keyboard custom-designed by the Appwrite team. It is optimized to improve developer productivity and is specially tuned to use with the Appwrite Console.
+
+The Appwriter uses icons from the Appwrite Console and Docs on specific keys, making memorizing keyboard shortcuts easier. For example, `G` then `D` is the shortcut for navigating to your project's databases, and the icon on the `D` key matches the icon for Appwrite Databases.
+
+## Specifications
+
+The box includes the Appwriter keyboard, a USB-C data/charging cable, a 2-in-1 keycap and switch puller, extra PC layout keycaps, and an information card with a QR code. The keyboard comes with Mac layout keys pre-installed for MacOS which can also be swapped with the included PC keycaps.
+
+|Specification|Details|
+|----|----|
+|Battery|3000mAh Rechargeable battery|
+|Design|75% Compact design keyboard|
+|Keycaps|84 Custom dye sublimated PBT keycaps|
+|Switches|Gateron g pro yellow pre-lubed switches|
+|Connections|Tri-mode connectivity (2.4Ghz wireless / bluetooth / wired USB-C)|
+|Case|Noise dampening case-foam|
+|RGB Backlight|16 Million colors & multiple lighting effects|
+|Switch Support|Hot-swappable switches with 3 & 5 pin|
+|N-Key|N-key rollover|
+
```

Comment:
Please add a section about how to change the RGB backlight effect. Pressing Fn + Home will cycle switch the effect, allowing you to cycle through them all.